crates/asm/subprotocols/bridge-v1/src/state/operator.rs

```diff
@@ -124,6 +126,27 @@ pub struct OperatorTable {
     /// The key is automatically computed when the operator table is created or
     /// updated, ensuring it always reflects the current active multisig participants.
     agg_key: BitcoinXOnlyPublicKey,
+
+    /// Historical N/N multisig scripts from previous operator set configurations.
+    ///
+    /// This vector tracks all previous P2TR scripts that were used when the operator set
+    /// changed (due to slashing or unstaking). These historical scripts are necessary to validate
+    /// slash transactions that reference stake connectors from previous operator configurations.
+    ///
+    /// # Use Case
+    ///
+    /// When multiple operators are slashed in sequence:
+    /// 1. First slash: Validated against current N/N multisig script, operator removed, new script
+    ///    computed
+    /// 2. Second slash: Must be validated against the previous N/N multisig script (before first
+    ///    slash) because the stake connector was created when that operator set was active
```

Comment:
why do we care about it? we validate against _all_ multisigs anyway. I think you're saying that we should not validate against the very last multisig, but:
1. is the vector deduplicated? That is, what if the very last multisig matches an old one
2. if someone is able to send an (illegitimate) slash tx with the very last (=current) multisig then we have other severe issues -- as the bridge multisig address is compromised.

Reply:
reading this again, you seems to focus on justifying why we validate against _all_ multisigs. If this is the intension, it's not clear to me why talking about "multiple operators are slashed in sequence" (what does in sequence mean? I read at as one immediately after the other)? there are other cases when this is required.